benchmark/rgw_benchmark_setup.yml

```diff
@@ -0,0 +1,29 @@
+---
+- hosts:
+    - mons
+  gather_facts: True
+  become: True
+  tasks:
+    - name: Create user rgwbench
+      command: "radosgw-admin user create --uid='rgwbench' --display-name='Rgw Benchmark'"
+      run_once: True
+    - name: Create subuser rgwbench:test
+      command: "radosgw-admin subuser create --uid='rgwbench' --subuser='rgwbench:test' --access=full --secret='{{ bench_rgw_user_password }}'"
+      run_once: True
+
+- hosts: benchmark_hosts
+  gather_facts: True
+  become: True
+  post_tasks:
+    ### Using the bench command of Hummingbird for benchmarking it's a single binary with no dependencies or additional libraries.
+    - name: Install benchmark software 
+      shell: 'wget {{ bench_rgw_hummingbird_url }} && chmod +x hummingbird && mv hummingbird /opt/ceph_bench/'
```

Comment:
Instead of using shell we can use get_url for the wget?  You can also use file for the chmod, and copy for the mv.  However that might not be necessary because get_url can get the url and place in a specific location and set the mode.

Reply:
Fixed